tools/src/bin/collect.rs

```diff
@@ -367,10 +332,9 @@ struct Options {
         required = false,
         num_args = 1,
         action = ArgAction::Set,
-        value_parser = BucketsValueParser::new(),
         help_heading = "VDAF Algorithm and Parameters"
     )]
-    buckets: Option<Buckets>,
+    buckets: Option<usize>,
```

Comment:
If we replace `buckets` with `length` elsewhere, we could eliminate this flag and simplify a bit.